cfme/tests/services/test_service_catalogs.py

```diff
@@ -148,7 +148,7 @@ def test_edit_catalog_after_deleting_provider(provider, catalog_item):
             catalog_item.update(changes)
         view = provider.create_view(EditCatalogItemView)
         view.flash.assert_message("'Environment/Cluster Name' is required", t='error')
-        view.flash.assert_message("'Network/Network' is required", t='error')
+        view.flash.assert_message("'Network/Virtual NIC Profile ID' is required", t='error')
```

Comment:
**REQUIRED:** Is this for VMware too

Reply:
@psav No, this branch of code is specific for RHV. AFAIK VMWare will let you update catalog item without specifying those fields.